src/panels/lovelace/cards/hui-light-card.ts

```diff
@@ -0,0 +1,313 @@
+        top: 0;
+        left: 0;
+        right: 0;
+        height: 100%;
+        text-align: center;
+        z-index: 15;
+      }
+      .icon-state {
+        display: block;
+        margin: auto;
+        width: 100%;
+        height: 100%;
+        transform: translate(0,25%);
+      }
+      #light {
+        margin: 0 auto;
+        padding-top: 16px;
+        padding-bottom: 16px;
+      }
+      #light .rs-bar.rs-transition.rs-first, .rs-bar.rs-transition.rs-second{
+        z-index: 20 !important;
+      }
+      #light .rs-range-color  {
+        background-color: var(--primary-color);
+      }
+      #light .rs-path-color  {
+          background-color: #d6d6d6;
+      }
+      #light .rs-handle  {
+          background-color: #FFF;
+          padding: 7px;
+          border: 2px solid #d6d6d6;
+      }
+      #light .rs-handle.rs-focus  {
+          border-color:var(--primary-color);
+      }
+      #light .rs-handle:after  {
+          border-color: var(--primary-color);
+          background-color: var(--primary-color);
+      }
+      #light .rs-border  {
+        border-color: transparent;
+      }
+      ha-icon {
+        margin: auto;
+        width: 76px;
+        height: 76px;
+        color: var(--paper-item-icon-color, #44739e);
+        cursor: pointer;
+      }
+      ha-icon[data-state=on] {
+        color: var(--paper-item-icon-active-color, #FDD835);
+      }
+      ha-icon[data-state=unavailable] {
+        color: var(--state-icon-unavailable-color);
+      }
+      .name {
+        padding-top: 40px;
+        font-size: 1.2rem;
+      }
+      .brightness {
+        font-size: 1.2rem;
+        position: absolute;
+        margin: 0 auto;
+        left: 50%;
+        top: 10%;
+        transform: translate(-50%);
+        opacity: 0;
+        transition: opacity .5s ease-in-out;
+        -moz-transition: opacity .5s ease-in-out;
+        -webkit-transition: opacity .5s ease-in-out;
+        cursor: pointer;
+        color: white;
+        text-shadow:
+          -1px -1px 0 #000,
+          1px -1px 0 #000,
+          -1px 1px 0 #000,
+          1px 1px 0 #000;
+      }
+      .show_brightness {
+        opacity: 1;
+      }
+      .not-found {
+        flex: 1;
+        background-color: yellow;
+        padding: 8px;
+      }
+    </style>
+    `;
+  }
+
+  private _dragEvent(e) {
+    this.shadowRoot!.querySelector(".brightness")!.innerHTML = e.value + "%";
+  }
+
+  private _showBrightness() {
+    clearTimeout(this.brightnessTimout);
+    this.shadowRoot!.querySelector(".brightness")!.classList.add(
+      "show_brightness"
+    );
+  }
+
+  private _hideBrightness() {
+    this.brightnessTimout = setTimeout(() => {
+      this.shadowRoot!.querySelector(".brightness")!.classList.remove(
+        "show_brightness"
+      );
+    }, 500);
+  }
+
+  private _setBrightness(e) {
+    this.hass!.callService("light", "turn_on", {
+      entity_id: this._config!.entity,
+      brightness_pct: e.value,
```

Comment:
Now I know why the gallery is flaky. We don't have support for `brightness_pct` in our demo hass for the gallery.

Reply:
How can I add that?

Reply:
Where did you look?

Reply:
You need to update the logic here https://github.com/home-assistant/home-assistant-polymer/blob/dev/gallery/src/data/entity.js#L56-L65